homeassistant/components/wink/__init__.py

```diff
@@ -176,8 +176,7 @@
 })
 
 WINK_COMPONENTS = [
-    'binary_sensor', 'sensor', 'light', 'switch', 'lock', 'cover', 'climate',
-    'fan', 'alarm_control_panel', 'scene'
+    'water_heater'
```

Comment:
This seems like a mistake.

Reply:
Opps yeah I was using that while testing so I didn't get all of my Wink devices pulled in. Good catch.